index.css

```diff
@@ -0,0 +1,225 @@
+html,body {
+    margin: 10px;
```

Comment:
Never set margin to body. It will lead to layout height issues. Better set it to 0; It is not a good practice to set layout styles in body. There could be font styles or other styles that should be nested by all children. But not layout specific styles.